superset/tags/models.py

```diff
@@ -181,6 +181,7 @@ def after_insert(
         session.add(tagged_object)
 
         session.commit()
+        session.close()
```

Comment:
The various SQLAlchemy hooks, `after_insert`, `after_delete`, etc. pass in the connection from which we instantiate a new session which was never closed. I'm really not sure what's wrong with simply using the Flask-SQLAlchemy session, i.e., `db.session`.